.github/workflows/build-docker-internal.yml

```
@@ -0,0 +1,62 @@
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Build and push internal image
+        uses: docker/build-push-action@v6
+        with:
+          build-args: |
+            BUILD_TYPE=released
+            FO_VERSION=${{ env.fo_version }}
+            PYTHON_VERSION=${{ env.pyver }}
+          context: .
+          # Not doing multi-platform builds as the intent of this
+          # is for security scanning. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)